cmds/gpt/gpt.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2016-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// gpt reads and writes GPT headers.
+//
+// Synopsis:
+//     gpt [-w] file
+//
+// Description:
+//     For -w, it reads a JSON formatted GPT from stdin, and writes 'file'
+//     which is usually a device. It writes both primary and secondary headers.
+//
+//     For -r, it writes the headers to stdout in JSON format.
```

Comment:
I do not see this flag in the flag list below.

Reply:
oops! 
fixed.